sdk/cosmos/azure-cosmos/test/__init__.py

```diff
@@ -0,0 +1 @@
+__path__ = __import__("pkgutil").extend_path(__path__, __name__)  # type: ignore
```

Comment:
There should not be an init file in the test directory. We don't want the test directory to become a Python module.
(This has additional implications for the collection in setup.py and manifest.in)

Reply:
I essentially wanted the cleanup file to have access to tests and azure/cosmos directories. Found a different way to do it.